sql-parser/src/main/java/io/crate/sql/tree/NullLiteral.java

```diff
@@ -21,8 +21,9 @@
 
 package io.crate.sql.tree;
 
-public class NullLiteral
-    extends Literal {
+public class NullLiteral extends Literal {
+    public static final NullLiteral INSTANCE = new NullLiteral();
```

Comment:
i couldn't do that because the constructor `NullLiteral()` is used outside this class